resources/js/views/global/Supercluster.vue

```diff
@@ -64,22 +65,36 @@ function createClusterIcon (feature, latlng)
 
 /**
  * On each feature, perform this action
+ *
+ * This is being performed whenever the user drags the map.
+ *
+ * Tranlsation should only occur when the user clicks on a point to open an image.
  */
 function onEachFeature (feature, layer)
 {
     if (! feature.properties.cluster)
     {
+        let a = feature.properties.result_string.split(',');
+        a.pop();
+
+        let z = '';
+        a.forEach(i => {
```

Comment:
Should use `reduce` instead of `forEach`

Reply:
We shouldn't loop here. The translation should only happen when the image is clicked and loaded.